### PR TITLE
fix: build config

### PR DESCRIPTION
**Pull request by simboonlong**
### pr updates
- pr builds should not be applying to master

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -5,8 +5,8 @@ name: CI
 on:
```
Comment (yehjxraymond) [inline .github/workflows/tests.yml]: Can delete this file, we are not using github actions here

File: .circleci/config.yml [hunk 2]
```diff
@@ -11,6 +11,6 @@ jobs:
       - run:
           name: test
           command: npm run test
-      - run:
-          name: deploy_staging
-          command: ./scripts/deploy_staging.sh
+      # - run:
```
Comment (yehjxraymond) [inline .circleci/config.yml]: Lol dont just delete, like that masters dont get deployed. Let is run on masters but not on other branches. See circles filtering:
https://circleci.com/docs/2.0/configuration-reference/#filters

File: .github/workflows/tests.yml (hunk 1 above)
Comment (simboonlong) [reply .github/workflows/tests.yml]: 👌

**Comment by simboonlong** [comment]
closing for this instead https://github.com/Open-Attestation/demo-identifier-resolver/pull/7

**Pull request closed without merging**